### PR TITLE
NTR - Add snippet loading according to docs

### PR DESCRIPTION
**Pull request by marcelbrode**

### CODE REVIEW

**Review by King-of-Babylon — APPROVED** [review]

File: src/Resources/administration/module/custom-module/snippet/en-GB.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "demoSnippet": "I'm a demo."
+
```
Comment (PaddyS) [inline src/Resources/administration/module/custom-module/snippet/en-GB.json]: Unnecessary line

**Review by PaddyS — CHANGES_REQUESTED** [review]

File: src/Resources/administration/module/custom-module/snippet/en-GB.json (hunk 1 above)
Comment (marcelbrode) [reply src/Resources/administration/module/custom-module/snippet/en-GB.json]: So you'd prefere an empty example-file? ok

Comment (PaddyS) [reply src/Resources/administration/module/custom-module/snippet/en-GB.json]: No. There was an unnecessary empty line.
```
{
    "demoSnippet": "I'm a demo." // Good line
    // Bad line
}
```

Comment (marcelbrode) [reply src/Resources/administration/module/custom-module/snippet/en-GB.json]: Oh :thinking:

**Review by PaddyS — APPROVED** [review]

**Pull request merged**